demo/Meter.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Meter Styling</title>
```

Comment:
The title of the other in-progress pages is `<title>CSS Chassis - Tables</title>`. Let's keep it consistent and include the `CSS Chassis`.